plugin/etcd_backup_plugin.go

```diff
@@ -0,0 +1,103 @@
+package plugin
+
+import (
+	ctx "context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.etcd.io/etcd/client/v3/snapshot"
+	"go.uber.org/zap"
+
+	"github.com/alibaba/sealer/logger"
+	"github.com/alibaba/sealer/utils/ssh"
+)
+
+type EtcdBackupPlugin struct {
+	name    string
+	backDir string
+}
+
+func (e EtcdBackupPlugin) Run(context Context, phase Phase) {
+	dialTimeout := 5 * time.Second
+	masterIP := context.Cluster.Spec.Masters.IPList[0]
+	if masterIP == "" {
+		logger.Error("Cluster does not exist \n")
+		os.Exit(1)
```

Comment:
return error

Reply:
Only exit directly when disaster happens. Otherwise let the caller decide whether to exit or not.

I think it is a principle that we should carry. 🐯